frontend/public/components/markdown-view.tsx

```diff
@@ -203,6 +203,8 @@ const IFrameMarkdownView: React.FC<InnerSyncMarkdownProps> = ({
     setLoaded(true);
   }, [updateDimensions]);
 
+  useResizeObserver(updateDimensions, frame);
```

Comment:
I think this is triggering the error:

> unhandled error: ResizeObserver loop limit exceeded 

If we update the frame height here, it triggers a ResizeObserver event since the frame size changed, which updates the height, etc.

Related to bug https://bugzilla.redhat.com/show_bug.cgi?id=2003120

Reply:
Normally we fail CI on any uncaught error. I think the reason this wasn't failing CI is because we had an uncaught error in our error handler... We assume `error` is defined in `window.onerror` when sometimes we just get `message`, so it blows up here before seeing `window.windowError`:

https://github.com/openshift/console/blob/master/frontend/public/components/app.jsx#L365